django/core/management/commands/shell.py

```diff
@@ -9,7 +9,12 @@
 
 
 class Command(BaseCommand):
-    help = "Runs a Python interactive interpreter. Tries to use IPython or bpython, if one of them is available."
+    help = (
+        "Runs Python code passed through standard input if any otherwise "
+        "runs an interactive interpreter, like the python CLI. "
+        "It tries to use Tries to use IPython or bpython, if one of them is available."
```

Comment:
I think the 'passed through standard input' is the most minor feature of the command, so it should come at the end of the help. Maybe take the original description and just add a sentence on the end about it instead?